pkg/gateway/api/api_grpc.go

```diff
@@ -393,11 +388,47 @@ func (s *grpcGatewayService) GetEvaluation(
 		)
 		return nil, ErrInternal
 	}
+	eval, err := s.findEvaluation(evaluations.Evaluations, req.FeatureId)
+	if err != nil {
+		return nil, err
+	}
 	return &gwproto.GetEvaluationResponse{
-		Evaluation: evaluations.Evaluations[0],
+		Evaluation: eval,
 	}, nil
 }
 
+func (s *grpcGatewayService) getTargetFeatures(fs []*featureproto.Feature, id string) ([]*featureproto.Feature, error) {
+	feature, err := s.findFeature(fs, id)
+	if err != nil {
+		return nil, err
+	}
+	if len(feature.Prerequisites) > 0 {
+		return fs, nil
```

Comment:
question:

Is it possible to select only the prerequisite feature flags?

Reply:
Talked at meeting.